openshift/ftest-openshift-assistant-template/src/test/java/org/arquillian/cube/openshift/standalone/HelloOpenShiftAssistantTemplateTest.java

```diff
@@ -0,0 +1,51 @@
+package org.arquillian.cube.openshift.standalone;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.arquillian.cube.openshift.impl.client.OpenShiftAssistant;
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.arquillian.cube.requirement.ArquillianConditionalRunner;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(ArquillianConditionalRunner.class)
+@RequiresOpenshift
+public class HelloOpenShiftAssistantTemplateTest {
+
+    @ArquillianResource
+    private OpenShiftAssistant openShiftAssistant;
+
+    @Test
+    public void should_inject_openshift_assistant() {
+        assertThat(openShiftAssistant).isNotNull();
+    }
+
+    @Test
+    public void should_apply_template_programmatically() throws IOException {
+
+        openShiftAssistant
+                .usingTemplate("https://gist.githubusercontent.com/lordofthejars/8781cacd4000a516695ad6c55b5815b3/raw/5151aeef0f5dd8823e2c581c3b7452f04a76af59/hello-template.yaml")
```

Comment:
This way we won't be able to test it when GH is down (which was the case few minutes ago). Does it matter we want it from some remote location?